### PR TITLE
HIV-454: Retrospective data entry settings revisions

### PR DESCRIPTION
**Pull request by maikofelix47**

### CODE REVIEW

**Review by kagai — DISMISSED** [review]

File: src/app/retrospective-data-entry/components/banner/retrospective-banner.component.html [hunk 1]
```diff
@@ -1,6 +1,14 @@
 <div class="retrospective-banner" *ngIf="retroEnabled">
   <retrospective-data-entry-settings (onSettingsChange)="toggleDialog($event)" [bannerMode]="true"></retrospective-data-entry-settings>
-  <p-dialog header="Retrospective Data Entry" [(visible)]="displayDialog" minHieght="300" position="center" modal="true">
-    <retrospective-data-entry-settings (onSettingsChange)="toggleDialog($event)" [modalMode]="true" [bannerMode]="false"></retrospective-data-entry-settings>
-  </p-dialog>
+  <div class="dialog-wrapper" *ngIf="displayDialog">
+    <div class="dialog-wrapper-content">
+      <p-dialog header="Retrospective Data Entry" 
+      [(visible)]="displayDialog" 
+      minHieght="300"
```
Comment (denniskigen) [inline src/app/retrospective-data-entry/components/banner/retrospective-banner.component.html]: Should this be `minHeight`?

Comment (maikofelix47) [reply src/app/retrospective-data-entry/components/banner/retrospective-banner.component.html]: I have removed it. Didn't seem to have any effect on the p-dialog. min height for later versions is passed via styles props

Comment (denniskigen) [reply src/app/retrospective-data-entry/components/banner/retrospective-banner.component.html]: Nice. lgtm

**Review by denniskigen — APPROVED** [review]

**Review by kipchubett — APPROVED** [review]
LGTM

**Pull request merged**